kube/Cargo.toml

```diff
@@ -48,6 +49,7 @@ features = []
 default = ["native-tls"]
 native-tls = ["openssl", "reqwest/native-tls"]
 rustls-tls = ["rustls", "reqwest/rustls-tls"]
+derive = ["kube-derive"]
```

Comment:
I don't have a nice way to publish crates that depend on in-source crates, and would need a lot of scripts to hack around this.

While, this might be possible, it also introduces confusion because both crates expose a `CustomResource` symbol. in kube: the manual CRD creator, in kube-derive: the automatic proc_macro.

Reply:
Well, I'd say that the naming of trait being `CustomResource` is correct. And would probably name the `CustomResource` struct a `ResourceDescriptor` or something similar.

Reply:
Yeah, that is fair. The duplicate naming is a bit confusing, and will try to update the names soon. Sorry for keeping this in the queue for now.

I appreciate the PR and will likely merge this at some point, it does require writing some better release scripts because of the within-repo crate dependency. So will wait until I have time to write that.

Reply:
This part is actually now done in master.